index.html

```diff
@@ -0,0 +1,542 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>OdinAdminDashboard</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <div class="grid">
+      <div class="sidebar">
+        <div class="logo-text">
+          <svg
+            style="width: 40px; height: 40px"
+            xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 24 24"
+          >
+            <title>ViewDashboard</title>
```

Comment:
title should be in plain english like "view dashboard"

Reply:
fixed it sir